### PR TITLE
refactor: added max length validation

### PR DESCRIPTION
**Pull request by craicoverflow**
https://issues.jboss.org/browse/AEROGEAR-9768

Added max length validation to the app name.

## Verification

1. Try to create an app > 70 characters. It should be prevented.

### CODE REVIEW

File: src/models/mobileapps/mobileapp.js [hunk 1]
```diff
@@ -43,8 +43,12 @@ export default class MobileApp {
     const value = this.getProperty(propertyName);
     if (value) {
       switch (propertyName) {
-        case PROPERTIES.NAME:
+        case PROPERTIES.NAME: {
+          if (value.length > 100) {
```
Comment (ziccardi) [inline src/models/mobileapps/mobileapp.js]: Better to make this a constants instead than hardcoding it here

File: src/components/create_client/EditMobileClientBaseClass.js [hunk 2]
```diff
@@ -7,7 +7,8 @@ import { VerticalFormField } from './VerticalFormField';
 
 export const LABEL_APPNAME = '* App Name';
 export const EXAMPLE_APPNAME = 'myapp';
-export const HELP_APPNAME = 'App name must only contain lowercase letters, numbers and dots.';
+export const HELP_APPNAME =
+  'App name can not exceed 100 characters, and must only contain lowercase letters, numbers and dots.';
```
Comment (ziccardi) [inline src/components/create_client/EditMobileClientBaseClass.js]: Better to make `100` a constants instead than hardcoding it here

**Comment by craicoverflow** [comment]
> Some points I have noted
> 
>     * No service will bind with more than 63 characters for the app name.
> 
>     * The name does not display correctly on the landing page cards if longer than 75 characters

These are pre-existing problems though, right?

**Comment by Boomatang** [comment]
The issue with the name been more 63 characters has been before with labels in the CR's. But the 75+ characters long name not showing correctly is only some thing I noticed today. There is no other issue created for it.

**Comment by craicoverflow** [comment]
@ziccardi I have changed the max length to be 70 characters. I believe this is more than sufficient. We could reduce it to be 50 to be more like Android and iOS, but I don't believe we need to. What do you think?

**Pull request merged**